scipy/stats/mstats_basic.py

```diff
@@ -460,7 +459,7 @@ def spearmanr(x, y, use_ties=True):
     (x, y) = (x.ravel(), y.ravel())
 
     m = ma.mask_or(ma.getmask(x), ma.getmask(y))
-    n -= m.sum()
+    n -= int(m.sum())
```

Comment:
While I realize there was a lot of small requests on you, here's hopefully the last one. Can you add a short comment here please? Otherwise the context is hidden in the test suite, and without context it's too easy to dismiss this type cast as extraneous, and remove it as a "cleanup". Esp given that most devs are on *nix, tests will not catch it.